vue-app/src/components/ClaimButton.vue

```diff
@@ -93,13 +94,14 @@ export default class ClaimButton extends Vue {
   claim() {
     this.$modal.show(
       ClaimModal,
-      { project: this.project },
-      {},
       {
-        closed: () => {
-          this.checkAllocation(this.$store.state.tally)
+        project: this.project,
+        claimed: () => {
+          // Optimistically update the claimed state
+          this.claimed = true
```

Comment:
We don't poll the data from the subgraph because it might not have been updated yet.